SkeletonableTableView/Sources/SkeletonableTableView/SkeletonableTableViewCell.swift

```diff
@@ -0,0 +1,107 @@
+    ///   - secondaryColor: The secondary color of gradient. Defaults to `.nil`.
+    ///   - transition: The style of the transition when the skeleton appears. Defaults to `.none`.
+    open func showGradientedSkeleton(baseColor: UIColor = SkeletonAppearance.default.tintColor,
+                                     secondaryColor: UIColor? = nil,
+                                     transition: SkeletonTransitionStyle = .none) {
+        let gradient = SkeletonGradient(baseColor: baseColor, secondaryColor: secondaryColor)
+        showGradientSkeleton(usingGradient: gradient, transition: transition)
+    }
+    
+    /// Shows the gradient animated skeleton on the SkeletonableTableViewCell.
+    ///
+    /// There is possibility to override function and change skeleton appearance configuration of  current cell
+    ///
+    /// - Parameters:
+    ///   - baseColor: The base color of gradient. Defaults to `SkeletonAppearance.default.tintColor`.
+    ///   - secondaryColor: The secondary color of gradient. Defaults to `.nil`.
+    ///   - gradientDirection: The animation direction of skeleton. Defaults to `.leftRight`.
+    ///   - duration: The duration of the animation. Defaults to `1.5`.
+    ///   - transition: The style of the transition when the skeleton appears. Defaults to `.none`.
+    open func showGradientedSkeletonAnimating(baseColor: UIColor = SkeletonAppearance.default.tintColor,
+                                              secondaryColor: UIColor? = nil,
+                                              gradientDirection: GradientDirection = .leftRight,
+                                              duration: Double = 1.5,
+                                              transition: SkeletonTransitionStyle = .none) {
+        let gradient = SkeletonGradient(baseColor: baseColor, secondaryColor: secondaryColor)
+        let animation = SkeletonAnimationBuilder().makeSlidingAnimation(withDirection: gradientDirection, duration: duration)
+        showAnimatedGradientSkeleton(usingGradient: gradient, animation: animation, transition: transition)
+    }
+    
+    /// Hides the skeleton on the SkeletonableTableViewCell.
+    ///
+    /// - Parameters:
+    ///   - transition: The style of the transition when the skeleton appears. Defaults to `.none`.
+    open func hideSkeleton(transition: SkeletonTransitionStyle = .none) {
+        hideSkeleton(reloadDataAfter: false, transition: transition)
+    }
+
+    /// Setup initial skeleton appearance configurations of cell
+    ///
+    /// If your child class is relized programatically you should call this method in init functions (child class) to make all needed subviews skeletonable. Otherwise do nothing
+    ///
+    /// This method makes skeletonable only subviews of contentView as default, but there is possibility to override function and change skeleton appearance configuration of specific views
```

Comment:
This method by default marks only subviews of contentView as skeletonable.
You can override it and tweak the marking behavior if you need